src/styles/ForecastSummary.css

```diff
@@ -0,0 +1,22 @@
+.forecast-summary{
+    padding: 15px;
+    border: 3px solid black;
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+    min-width: 170px;
+}
+.forecast-summary > *{
+    padding: 10px;
+}
+.forecast-summary__date{
+    text-align: center;
+    font-size: 20px;
+}
+.forecast-summary__icon{
+    text-align: center;
+    font-size: 70px;
+}
+.forecast-summary__temperature-max, .forecast-summary__description{
```

Comment:
Convention is to list each class on its own line:
```
.forecast-summary__temperature-max,
.forecast-summary__description {
```

Also, it would be good to leave a clear line between each class declaration to make the file a bit easier to read